src/apollo/zimbra-in-memory-cache.ts

```diff
@@ -40,7 +40,7 @@ const dataIdFromObject = (object: any) => {
 			if (object.part && object.part !== null) {
 				return `${defaultDataIdFromObject(object)}:${object.part}`;
 			}
-			return defaultDataIdFromObject(object);
+			return `${defaultDataIdFromObject(object)}`;
```

Comment:
do we need this?

Reply:
No.
Previous changes are not require and there is nothing change hence Dropping the PR.